### ISSUE TITLE
Terminal image support (Kitty graphics protocol, Sixel, etc)

### ISSUE THREAD

**Issue by rsmirnov90** [gui-cs/Terminal.Gui#3005]
Has anyone considered adding support for rendering an image in the text terminal? 

I can think of many scenarios where this could come in useful, and would love to work on something like this myself though I likely wouldn't even know where to start.

**Comment by tznind** [comment]
There has been consideration but never more than that.

See this discussion for example (which contains link to example code):
https://github.com/gui-cs/Terminal.Gui/discussions/1265

If you are motivated to give it a go then the best place to start would probably be getting a standalone console app to produce a sixel image.  Then we can help integrate it into the driver layer or a `View`.

**Comment by Nutzzz** [comment]
FYI, VT support has recently been much improved in Windows Terminal/ConPTY, and Sixel is now working in Windows Terminal (Canary only for now)!  See https://github.com/microsoft/terminal/pull/17510 and https://github.com/microsoft/terminal/pull/17581

![image](https://github.com/user-attachments/assets/c480cc56-e4c2-4db2-a88c-1f774020395b)

**Comment by BDisp** [comment]
It's necessary install Sixel?

**Comment by Nutzzz** [comment]
> It's necessary install Sixel?

For the above test, nothing to install was necessary (other than WT Canary).

**Comment by BDisp** [comment]
My WT doesn't show anything about Canary in the version.

**Comment by tznind** [comment]
Here is a link to the blog post, which contains a link to GitHub repo including installers

https://devblogs.microsoft.com/commandline/introducing-windows-terminal-canary/

Presume canary is just the 'early adopter' channel.  So this feature will come later to Windows Terminal core releases too.

**Comment by Nutzzz** [comment]
> Presume canary is just the 'early adopter' channel. So this feature will come later to Windows Terminal core releases too.

Correct.  You can download the latest Canary installer here: https://aka.ms/terminal-canary-installer or just wait until the end of the month (estimated) to update your WT from the Microsoft Store.
